roles/solrcloud/tasks/config.yml

```diff
@@ -1,134 +1,88 @@
 ---
-- name: Solrcloud | Check for solr znode
-  ansible.builtin.command: >
-    {{ solr_installation }}/bin/solr zk ls /
-    -z {{ solr_zookeeper_hosts_string }}
-  register: znode_check
-  ignore_errors: true
-  changed_when: false
+# Configure SolrCloud
 
-- name: Solrcloud | Create zk chroot / solr znode using bin/solr
-  ansible.builtin.command: >
-    {{ solr_installation }}/bin/solr zk mkroot {{ solr_znode }}
-    -z {{ solr_zookeeper_hosts_string }}
-  when: solr_znode not in znode_check.stdout_lines and solr_znode != ''
-  changed_when: false
+- name: Solrcloud | Debug Solr variables
+  ansible.builtin.debug:
+    msg:
+      - "solr_installation: {{ solr_installation }}"
+      - "solr_bin: {{ solr_bin }}"
+      - "solr_cloud_download_version: {{ solr_cloud_download_version }}"
+      - "solr_znode: {{ solr_znode | default('undefined') }}"
```

Comment:
Do we want to keep this? Or was this a tool that helped debug and now it can go?